### DISCUSSION TITLE
Ubuntu 22.04.3 - libraries not found

### DISCUSSION THREAD

**Discussion by andysbaran** [mesa/mesa#1833] [Q&A]
Hello,
I installed SDK (the current version) for linux and after completing all steps and having all prerequisites installed, when running e.g. 'gcc' I get an error message that ......./x86_64-pc-linux-gnu/bin/ld: cannot find crt1.o and crti.o
I searched for them on my machine and I find them both in /usr/lib/x86_64-linux.gnu
I have Ubuntu 22.04.3 LTS.
This is not the MESA installation yet but I want to have this part completed before I hit on the next step.
Thanks in advance.

**Comment by rht** [comment]
This is Mesa agent-based model library for computational science, not https://en.wikipedia.org/wiki/Mesa_(computer_graphics). You should go to https://gitlab.freedesktop.org/mesa/mesa/ instead.
